.buildkite/pipelines/pull_request/base.yml

```diff
@@ -132,7 +132,7 @@ steps:
   - command: .buildkite/scripts/steps/functional/defend_workflows.sh
     label: 'Defend Workflows Cypress Tests'
     agents:
-      queue: n2-4-virt
```

Comment:
Can we make this is reverted after testing is done & before merging?  The packer_cache.sh changes will be pulled in to n2-4-virt after merging.

Reply:
Sure, will revert it once the current CI finishes. Thanks for the reminder!

Reply:
@jbudz done!